src/Compiler/ES/ES.idr

```diff
@@ -377,6 +377,7 @@ jsPrim (NS _ (UN "prim__os")) [] =
     let oscalc = "(o => o === 'linux'?'unix':o==='win32'?'windows':o)"
     sysos <- addConstToPreamble "sysos" (oscalc ++ "(" ++ os ++ ".platform())")
     pure sysos
+jsPrim (NS _ (UN "void")) [_, _] = jsCrashExp $ jsString $ "Error: Executed 'void'"
```

Comment:
Thank you for the review :)
```suggestion
jsPrim (NS _ (UN "void")) [_, _] = jsCrashExp $ jsString $ "Error: Executed 'prim__void'"
```
is this the change you meant?

Reply:
https://github.com/idris-lang/Idris2/blob/f33feecbea82aee09883eed0f50b0f68216085ab/src/Compiler/Scheme/Common.idr#L448
I got the inspiration from here, should both be changed?

Reply:
No I meant changing `prelude` to match the convention of calling `%extern` functions
`prim__SOMETHING`. You'd then have to change the left-hand side here to match on
`jsPrim (NS _ (UN "prim__void"))`  instead of `void` but the RHS can stay the same.

Reply:
Humm, that way I would have to change Prelude, no?
https://github.com/idris-lang/Idris2/blob/e93fba3952649762d8d165124c0888756547dacf/libs/prelude/Prelude/Uninhabited.idr#L15-L18

Reply:
I need some help, I get this error when I boostrap:
Uncaught error: INTERNAL ERROR: Can't compile unknown external primitive Prelude.Uninhabited.prim__void

How should I go about this?

Reply:
I have pushed a patch. Let's see if it works.